README.md

```diff
@@ -41,6 +51,8 @@ To get started add the `GoogleService-Info.plist` and the `google-services.json`
 </ItemGroup>
 ```
 
+Be sure to change the `TargetFramework` condition to the correct value for your target versions (i.e. `net6.0-*`, `net7.0-*`, `net8.0-*`, etc.)
```

Comment:
I believe we can remove this, and update the section code above to be agnostic to the .NET version. So, it can be like our friend Gerald suggested:


```xml
<ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">
      <BundleResource Include="GoogleService-Info.plist" Link="GoogleService-Info.plist" />
</ItemGroup>

<ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">
      <GoogleServicesJson Include="google-services.json" />
</ItemGroup>
```